switch/mupdf/PKGBUILD

```diff
@@ -0,0 +1,51 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: Julian Uy <uyjulian@gmail.com>
+
+pkgname=switch-mupdf
+pkgver=1.16.1
+pkgrel=1
+pkgdesc='MuPDF is a lightweight PDF, XPS, and E-book viewer.'
+arch=('any')
+url='https://www.mupdf.com/'
+license=('AGPL')
+options=(!strip libtool staticlibs)
+depends=("switch-freetype"
+         "switch-libjpeg-turbo"
+         "switch-zlib"
+)
+makedepends=('devkitpro-pkgbuild-helpers')
+source=("https://www.mupdf.com/downloads/archive/mupdf-$pkgver-source.tar.xz" "mupdf.patch")
+sha256sums=(
+  '6fe78184bd5208f9595e4d7f92bc8df50af30fbe8e2c1298b581c84945f2f5da'
+  '71543c287162cf71d7dff9a6d6ea1fa9b16c9f8d6975ebf66a9d154c986f4f7f'
+)
+
+build() {
+  cd mupdf-$pkgver-source
+
+  patch --strip=1 --input=$srcdir/mupdf.patch
+
+  make generate
+
+  source /opt/devkitpro/switchvars.sh
+
+  make build=release \
+    CC=${TOOL_PREFIX}gcc CXX=${TOOL_PREFIX}g++ AR=${TOOL_PREFIX}ar \
+    LD=${TOOL_PREFIX}ld RANLIB=${TOOL_PREFIX}ranlib \
+    XCFLAGS="${CPPFLAGS} '-Dtimegm(x)=-1'" \
+    OS=switch \
```

Comment:
Would it not be better to patch Makerules and do all this configuration for switch there?

http://git.ghostscript.com/?p=mupdf.git;a=blob;f=Makerules;h=d2e585daa0708ea23b961bf39dc407e9733744f1;hb=refs/heads/master#l168